lib/pages/app_bar_view.dart

```diff
@@ -0,0 +1,57 @@
+part of 'home_page.dart';
+
+extension AppBarView on _HomePageState {
```

Comment:
Let's put extensions on a special folder.

Reply:
This is part of home page.